### PR TITLE
Announce Camel Quarkus 2.8.0-M1

### PR DESCRIPTION
**Pull request by aldettinger**
Please do not merge before camel-quarkus 2.8.0-M1 has been integrated in quarkus-platform next week.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by jamesnetherton — APPROVED** [review]

**Review by zbendhiba — APPROVED** [review]

**Comment by oscerd** [comment]
@zregvart do you have any idea about the sitemap and security links checks failing in the PR check?

File: content/blog/2022/03/camel-quarkus-release-2.8.0-M1/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+summary: "Camel Quarkus 2.8.0-M1 brings Quarkus 2.7.3.Final, Camel 3.15.0"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the 2.8.0-M1 release of Camel Quarkus.
+It brings Camel 3.15.0 and Quarkus 2.7.3.Final.
+Many thanks to all contributors and issue reporters!
+
+Users are highly encouraged to test this milestone release in order to early catch possible flaws before the final Camel Quarkus 2.8.0 release.
```
Comment (zbendhiba) [inline content/blog/2022/03/camel-quarkus-release-2.8.0-M1/index.md:16]: awesome !!
Many thanks @aldettinger

**Comment by aldettinger** [comment]
Closing as the X.Y.Z-M1 version style is not supported. We'll publish a 2.8.0 announcement that could aggregate 2.8.0-M1 and 2.8.0.

**Comment by zregvart** [comment]
> @zregvart do you have any idea about the sitemap and security links checks failing in the PR check?

The root cause seems to be this error:
```
➤ YN0000: ERROR 2022/03/17 12:33:30 render of "page" failed: execute of template failed: template: release-note/single.html:13:7: executing "release-note/single.html" at <partial (printf "releases/%s.html" (lower (replace .Params.category " " "-"))) .>: error calling partial: execute of template failed: template: partials/releases/camel-quarkus.html:1:3: executing "partials/releases/camel-quarkus.html" at <partial "releases/downloads.html" (dict "Category" (index (index $.Site.Data "release-categories") "camel-quarkus") "Pages" $.Site.Pages "Version" .Params.version)>: error calling partial: execute of template failed: template: partials/releases/downloads.html:20:33: executing "partials/releases/downloads.html" at <partial "releases/version_sort.html" $category_releases>: error calling partial: "/home/runner/work/camel-website/camel-website/layouts/partials/releases/version_sort.html:10:55": execute of template failed: template: partials/releases/version_sort.html:10:55: executing "partials/releases/version_sort.html" at <int $version_part>: error calling int: unable to cast "0-M1" of type string to int
```

So I'm assuming the pages were not built correctly.

**Comment by oscerd** [comment]
On the other PR, the one related to camel stats, there are different errors. But I need to check again.

**Pull request closed without merging**